pac4j-http/src/test/java/org/pac4j/http/credentials/password/BasicSaltedSha512PasswordEncoderTests.java

```diff
@@ -21,12 +21,12 @@
 import static org.junit.Assert.assertEquals;
 
 /**
- * This class tests the {@link SaltedSha512PasswordEncoder}.
+ * This class tests the {@link BasicSaltedSha512PasswordEncoder}.
  * 
  * @author Jerome Leleu
  * @since 1.8.0
  */
-public final class SaltedSha512PasswordEncoderTests {
+public final class BasicSaltedSha512PasswordEncoderTests {
```

Comment:
Not related to this PR, renaming this default password encoder to make it as a "Basic" one.